tdrs-backend/tdpservice/users/serializers.py

```diff
@@ -10,7 +10,6 @@
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
-logger.addHandler(logging.StreamHandler())
```

Comment:
This was causing some duplicated logs, and is unneeded here as it is already declared in `settings.py`

Reply:
good catch!